### PR TITLE
suck it bitch

### PR DESCRIPTION
**Pull request by Alkesst**
- /arkangel reply with an amazing audio

### CODE REVIEW

File: api/_bot/dev.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import 'regenerator-runtime'
```
Comment (Alkesst) [inline api/_bot/dev.js]: useful when testing async methods/funcs

File: package.json [hunk 2]
```diff
@@ -67,6 +67,7 @@
     "env-cmd": "^10.0.1",
     "parcel-bundler": "^1.11.0",
     "parcel-plugin-static-files-copy": "^2.2.1",
-    "parcel-plugin-sw-cache": "^0.3.1"
+    "parcel-plugin-sw-cache": "^0.3.1",
+    "prettier": "^1.19.1"
```
Comment (Alkesst) [inline package.json]: prettier was not in the dev-dependences

File: api/_bot/dev.js (hunk 1 above)
Comment (melchor629) [reply api/_bot/dev.js]: changed this by using a custom configuration of babel

File: api/_bot/.babelrc [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (melchor629) [inline api/_bot/.babelrc]: Added bot-level `.babelrc` so `babel-watch` will generate code for node instead code for the browser

Comment (Neko250) [reply api/_bot/.babelrc]: 🎉

**Pull request merged**